.travis.yml

```diff
@@ -5,5 +5,5 @@ cache:
   directories:
     - node_modules
 before_script: 
-- cd node_modules/truffle && npm install solc@0.5.0 && cd ../..
```

Comment:
Instead of this, I would prefer Anxo's approach https://github.com/gnosis/dx-contracts/blob/master/package.json#L32

Reply:
Having it only in the .travis.yml means users will not have it working directly after git clone, npm i...

Reply:
I would do that in a separate PR (or maybe in the npm package one by you -> #89 )